OWNERS_ALIASES

```diff
@@ -122,6 +122,23 @@ aliases:
     - mikedanese
     - WanLinghao
 
+  # SIG Release
+  sig-release-approvers:
```

Comment:
Maybe I missed it, but looks like `sig-release-approvers` is not being used anywhere?

Reply:
@nikhita -- it's not currently used anywhere, but we may decide to later.